backend/app/api/routes/collections.py

```
@@ -178,14 +183,27 @@ def _backout(crud: OpenAIAssistantCrud, assistant_id: str):
         )
 
 
+def mark_collection_failed(session, user_id, collection_id, reason: str):
```

Ruff (0.12.2)

Unused function argument: `reason`

(ARG001)